app/models/concerns/smooch_search.rb

```diff
@@ -21,7 +21,7 @@ def search(app_id, uid, language, message, team_id, workflow, provider = nil)
 
         # Search for explainers if fact-checks were not found
         if reports.empty? && query['type'] == 'text'
-          explainers = self.search_for_explainers(uid, query['text'], team_id, language).first(3).select{ |explainer| explainer.as_tipline_search_result.should_send_in_language?(language) }
+          explainers = self.search_for_explainers(uid, query['text'], team_id, language).select{ |explainer| explainer.as_tipline_search_result.should_send_in_language?(language) }
```

Comment:
This line is only called by the tipline search. So here we could pass to `search_for_explainers` the configured value for the maximum number of results for tipline search (3 by default).

Reply:
Done